mw-webapp/src/utils/DateUtils.ts

```diff
@@ -88,4 +88,11 @@ export class DateUtils {
     return a.getTime() - b.getTime();
   }
 
+  /**
+   * Dates ascendant sorter
+   */
+  public static datesDESCSorter(a: Date, b: Date) {
```

Comment:
it is not a sorter it is Comparator (compare)

Reply:
Then need rename and datesASCSorter

Reply:
done